Sources_v3/WebSocketClient/Events/ConnectionEvents.swift

```diff
@@ -32,3 +32,15 @@ public struct HealthCheckEvent: ConnectionEvent, EventWithPayload {
         )
     }
 }
+
+/// Emitted when `Client` changes it's connection status. You can listen to this event and indicate the different connection
+/// states in the UI (banners like "Offline", "Reconnecting"", etc.).
+public struct ConnectionStatusUpdated: Event {
+    /// The current connection status of `Client`
+    public var connectionStatus: ChatClientConnectionStatus {
```

Comment:
I think could be `lazy var` to avoid recreating of `ChatClientConnectionStatus`.

Reply:
This has to be a computed variable, otherwise it'd just cashed the first "state" value and then never change again.